controllers/dashboard_controller.go

```diff
@@ -56,8 +57,10 @@ import (
 )
 
 const (
-	initialSyncDelay = "10s"
-	syncBatchSize    = 100
+	initialSyncDelay               = "10s"
+	syncBatchSize                  = 100
+	conditionDashboardSynchronized = "DashboardSynchronized"
+	conditionErrorFetchingInstance = "ErrFetchingInstances"
```

Comment:
Not sure if this should be a condition as the error is not fixable by the end user - only by kubernetes/operator admins

Reply:
Ok I removed this one